src/components/podcast/Tile.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+/**
+ * Component for displaying the tile of a podcast
+ * @param {*} podcast podcast object
+ */
+const PodcastTile = ({ podcast }) => {
```

Comment:
IMO an implicit return reads cleaner for stateless components like this, but YMMV:

```
const PodcastTile = ({ podcast }) =>
    <div>
      <a href={podcast.name}>{podcast.title}</a>
    </div>
```

Another consideration (here it def doesn't matter), is when to avoid object props like podcast here.

A much more reusable component would be one that takes `href` and `title` props.

ie:
```
<PodcastTitle title={podcast.title} href={podcast.name} />
```

or if you keep use `name` instead of `href` as a prop you could do:
```
<PodcastTitle {...podcast} />
```
... I like shortcuts like this but I generally think the above is better because it's much clearer what's getting used, and what it is being used for.

The general guideline I follow is if I'm creating a "container" or "view" component, it's likely that it'll _need_ to accept entire objects as props (eg: when injecting something from a global state store)... everything else (especially components that are likely to be reused) I lean towards named props. All that said, if you're passing more than 5 or 6 props that are all part of the same model, it's time to break this rule. eg: a `PodcastCard` component would almost certainly need quite a bit of information that it wouldn't make sense to split out into props.